Gruntfile.js

```diff
@@ -261,11 +261,14 @@ module.exports = (grunt) => {
     ...pick(clone(esbuildOptionBags.default), [
       'define', 'bundle'
     ]),
-    // format: 'esm',
+    // Format must be 'iife' because we don't want 'import' in the output
     format: 'iife',
-    // banner: {
+    banner: {
+      // This banner makes contracts easy to detect and harmless in the event
+      // of accidental execution
+      js: 'for(;;)"use shelter";'
```

Comment:
Please update this comment to say that the reason this is being added is to prevent people from uploading malicious contracts to our servers and then loading them elsewhere (e.g. via XSS or whatever).